endpoint.go

```diff
@@ -431,6 +431,39 @@ func (ep *endpoint) sbJoin(sbox Sandbox, options ...EndpointOption) error {
 	return sb.clearDefaultGW()
 }
 
+func (ep *endpoint) rename(name string) error {
+	n := ep.getNetwork()
+	if n == nil {
+		return fmt.Errorf("network not connected for ep %q\n", ep.name)
+	}
+
+	n.getController().Lock()
+	nw, ok := n.getController().nmap[n.ID()]
```

Comment:
I was following the convention in rest of the code..but I agree that a longer will be good for better readability. will change it.